internal/pkg/agent/application/coordinator/coordinator_test.go

```diff
@@ -335,7 +335,6 @@ func TestCoordinatorShutdownErrorOneResponse(t *testing.T) {
 }
 
 func TestCoordinatorShutdownErrorAllResponses(t *testing.T) {
-	CoordinatorShutdownTimeout = time.Millisecond
```

Comment:
Doesn't this make the value of the shutdown timeout dependent on the order in which tests are executed?

Reply:
nice catch, I added these back and set to what the default is during normal operations